modules/kubernetes-addons/kube-prometheus-stack/values.yaml

```diff
@@ -0,0 +1,151 @@
+## Create default rules for monitoring the cluster
+## Disable rules for unreachable components
+##
+defaultRules:
+  create: true
+  rules:
+    etcd: false
+    kubeScheduler: false
+
+## Disable component scraping for the kube controller manager, etcd, and kube-scheduler
+## These components are not reachable on EKS
+##
+kubeControllerManager:
+  enabled: false
+kubeEtcd:
+  enabled: false
+kubeScheduler:
+  enabled: false
+
+
+############################################
+## Below are the values.yaml file entries to control the images used by the kube-prometheus-stack
```

Comment:
I think we can remove all this thats commented out and let users refer to the official chart and its values.yaml file for customization